test/test231.js

```diff
@@ -62,7 +62,7 @@ describe('Test 231 NIST SQL Example', function() {
                     (ID INTEGER REFERENCES STATION(ID), \
                     MONTH INTEGER CHECK (MONTH BETWEEN 1 AND 12), \
                     TEMP_F REAL CHECK (TEMP_F BETWEEN -80 AND 150), \
-                    RAIN_I REAL CHECK (RAIN_I BETWEEN 0 AND 100), \
+                    RAIN_I REAL CHECK (VALUE->RAIN_I BETWEEN 0 AND 100), \

```

Comment:
This change can not be parsed at the moment: https://travis-ci.org/agershun/alasql/jobs/157699517#L816

Did you push the last version of the `dist/` folder?